gateserver/controller_api.py

```diff
@@ -0,0 +1,43 @@
+from . import controller_protocol as p
```

Comment:
Tak som to nemyslel. Bud `from . import controller_protocol`, alebo este lepsie `from .controller_protocol import konkretne, veci, co, treba`.